Apps/Admin/Client/Store/SupportUser/SupportUserReducers.cs

```diff
@@ -0,0 +1,114 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.Admin.Client.Store.SupportUser
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using Fluxor;
+    using HealthGateway.Admin.Client.Models;
+
+    /// <summary>
+    /// The set of reducers for the feature.
+    /// </summary>
+    public static class SupportUserReducers
+    {
+        /// <summary>
+        /// The Reducer for loading the support user.
+        /// </summary>
+        /// <param name="state">The support user state.</param>
+        /// <returns>The new state.</returns>
+        [ReducerMethod(typeof(SupportUserActions.LoadAction))]
+        public static SupportUserState ReduceLoadAction(SupportUserState state)
+        {
+            return state with
+            {
+                IsLoading = true,
+            };
+        }
+
+        /// <summary>
+        /// The Reducer for the load success action.
+        /// </summary>
+        /// <param name="state">The support user state.</param>
+        /// <param name="action">The load success action.</param>
+        /// <returns>The new state.</returns>
+        [ReducerMethod]
+        public static SupportUserState ReduceLoadSuccessAction(SupportUserState state, SupportUserActions.LoadSuccessAction action)
+        {
+            return state with
+            {
+                IsLoading = false,
+                Result = action.Data,
+                Error = null,
+                WarningMessage = action.Data.ResultError?.ResultMessage,
+                Data = action.Data.ResourcePayload?.Select(u => new ExtendedSupportUser(u)).ToList(),
+            };
+        }
+
+        /// <summary>
+        /// The Reducer for the fail action.
+        /// </summary>
+        /// <param name="state">The message verification state.</param>
+        /// <param name="action">The load fail action.</param>
+        /// <returns>The new state.</returns>
+        [ReducerMethod]
+        public static SupportUserState ReduceLoadFailAction(SupportUserState state, SupportUserActions.LoadFailAction action)
+        {
+            return state with
+            {
+                IsLoading = false,
+                Error = action.Error,
+            };
+        }
+
+        /// <summary>
+        /// The Reducer for the reset state action.
+        /// </summary>
+        /// <param name="state">The support user state.</param>
+        /// <returns>The empty state.</returns>
+        [ReducerMethod(typeof(SupportUserActions.ResetStateAction))]
+        public static SupportUserState ReduceResetStateAction(SupportUserState state)
+        {
+            return state with
+            {
+                IsLoading = false,
+                Result = null,
+                Error = null,
+                Data = null,
+            };
+        }
```

Comment:
The WarningMessage property should also be reset.

Reply:
Fixed.